DEVELOPMENT.md

```diff
@@ -0,0 +1,143 @@
+# Development
+
+Please make sure to read the [Contributing guideline](./CONTRIBUTING.md) first.
+
+## How to manage the BigAnimal Bearer Token?
+
+As described in the [Readme](./README.md#getting-an-api-token) file, you can use the `get-token.sh` script to fetch a Bearer Token.
+
+Another common practice is using the [Biganimal CLI](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/).
+
+### Using BA CLI to help initializing Provider credentials
+
+1. [Install the BA CLI](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/#installing-the-cli) and [jq - Command Line JSON Processor ](https://stedolan.github.io/jq/).
+1. [Authenticate as a valid user and create a credential](https://www.enterprisedb.com/docs/biganimal/latest/reference/cli/#installing-the-cli). This command will direct you to your browser.
+```shell
+biganimal create-credential \
+  --name "ba-user1" \
+  --address "portal.biganimal.com" \
+  --port "443"
+```
+3. Add the following bash functions to your shellrc file (For example: `.bashrc` if you're using bash, `.zshrc` if you're using ZSH) and start a new shell.
+```bash
+ba_api_get_call () {
+	endpoint=$1
+	curl -s --request GET --header "content-type: application/json" --header "authorization: Bearer $BA_BEARER_TOKEN" --url "$BA_API_URI$endpoint"
+}
+
+ba_get_proj_id () {
```

Comment:
The organization ID is set as the default project, therefore I believe the most fitting function name would be `ba_get_default_proj_id`.

Reply:
That's a good idea, thanks 👍